src/main/resources/native/android/AndroidManifest.xml

```diff
@@ -3,8 +3,7 @@
     <supports-screens a:xlargeScreens="true"/>
     <uses-sdk a:minSdkVersion="21" a:targetSdkVersion="28"/>
     <uses-permission a:name="android.permission.INTERNET"/>
-    <uses-permission a:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-permission a:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <!-- PERMISSIONS -->
```

Comment:
I assume the third party jar (usually Attach) will have a file with lines like:
```
<uses-permission a:name="android.permission.CAMERA"/>
```
right?

However, I'd say it should only hold the permission itself:

```
android.permission.CAMERA
```
And Substrate will provide the format: `<uses-permission a:name="` + line + `"/>`

This has the advantage of not adding a dependency of the namespace (`a`) to those jars.

Reply:
Done!

Reply:
Ok, but I don't see the commit?

Reply:
Oops, `git push` failed :)

I have pushed it now.